### ISSUE TITLE
Config.json file for custom relation extraction.

### ISSUE THREAD

**Issue by ahmadaii** [studio-ousia/luke#114]
I have prepared a Tacred like dataset for relation extraction with different relation labels and it gets trained successfully with 98 accuracy.
Unfortunately, config.json file is not in model output directory. Do I need to create manually?
How do I create this?

**Comment by ikuyamada** [comment]
Hi @ahmadaii,
If I correctly remember, our code does not use a file named `config.json`.

**Comment by elonmusk-01** [comment]
@ikuyamada I understand this. But when I wish to use the fine-tuned model for making predictions using the transformer library it asks for some other files like config.json of them. I have found fine-tuned luke models on the Huggingface hub having all the necessary files. My question is again the same how can I use pytorch_model.bin and not any other file to make predictions after fine-tuning it on custom dataset.

**Comment by elonmusk-01** [comment]
Thanks @ikuyamada I will give a try on allennlp. But I have spent a lot of time and cloud resources so I would like to use my fine-tuned re model on tacred like dataset. Meanwhile can you please tell which library can be used to made predictions from the model I fine-tuned using the non-allennlp based relation_classificatio task.

**Comment by elonmusk-01** [comment]
![Screenshot from 2022-03-03 11-57-52](https://user-images.githubusercontent.com/98889224/157497635-456cf943-eac9-4d36-a4c2-0337e8279c9b.png)

**Comment by elonmusk-01** [comment]
Isn't it scary after fine-tuning if we are still not ready to use this model in any library?

**Comment by ikuyamada** [comment]
@elonmusk-01  As the example code is based on PyTorch, you can run inference using it. Please refer to the documentation of PyTorch for the details.
